<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>sight_add</class>
 <widget class="QDialog" name="sight_add">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>599</width>
    <height>393</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>관광지 등록</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>20</y>
     <width>381</width>
     <height>321</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QLineEdit" name="name_txt"/>
    </item>
    <item>
     <widget class="QLineEdit" name="type_txt"/>
    </item>
    <item>
     <widget class="QLineEdit" name="addr_txt"/>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>320</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 0, 0);
color: rgb(255, 255, 255);
border-radius: 5px;</string>
   </property>
   <property name="text">
    <string>추가</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>50</y>
     <width>99</width>
     <height>82</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 15pt &quot;ori1Uni&quot;;</string>
   </property>
   <property name="text">
    <string>장소명</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>130</y>
     <width>61</width>
     <height>101</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 15pt &quot;ori1Uni&quot;;</string>
   </property>
   <property name="text">
    <string>업종</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>200</y>
     <width>81</width>
     <height>141</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 15pt &quot;ori1Uni&quot;;</string>
   </property>
   <property name="text">
    <string>주소</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
